### DISCUSSION TITLE
Unable to update / delete payment method

### DISCUSSION THREAD

**Discussion by bhargavaman** [community/community#45108] [Other Feature Feedback, Questions, & Ideas]
On the update payment page, I currently have a paypal set up that I'd like to replace with a credit card. 

But whenever I click on "Edit" it just sits there and nothing happens. 

I just wanna pay for Copilot :(

--- EDIT ---
Upon checking the console log, here's what I found:
![image](https://user-images.githubusercontent.com/5390139/214444858-36240ae3-da3e-4da1-aecd-2f91d15376df.png)

The first few errors occur during the page refresh. When I click the Edit button, I get the final "Cannot set properties of null" error that's at the bottom. 

Hope this helps!

**Comment by jge162** [comment]
possibly a problem with your browser. Try clearing your browser's cache and cookies, or try accessing the website using a different browser. e.g. go from Chrome to FireFox or Safari...

**Reply by bhargavaman** [reply]
Thanks for replying. I thought as much, but I've tried it across multiple browsers, across multiple machines (including private browsing)

That doesn't seem to be it, the issue still persists.

**Reply by jge162** [reply]
Edit button = broken atm. It’s clearly not working. Sorry to hear.

**Reply by jge162** [reply]
> Thanks for replying. I thought as much, but I've tried it across multiple browsers, across multiple machines (including private browsing)
> 
> That doesn't seem to be it, the issue still persists.

Here is link to submit for help https://support.github.com/request

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @bhargavaman ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!

**Reply by bhargavaman** [reply]
My apologies, and thanks!
